android/src/org/ppsspp/ppsspp/LocationHelper.java

```diff
@@ -74,17 +73,7 @@ public void onNmeaReceived(long timestamp, String nmea) {
 							onNmea(nmea);
 						}
 					};
-
-					// Use reflection to work around a bug in the Android 29 SDK.
-					// https://stackoverflow.com/questions/57975969/accessing-nmea-on-android-api-level-24-when-compiled-for-target-api-level-29
-					try {
```

Comment:
Hm, how come we can remove this?

Reply:
Oh wait, never mind, apparently it was indeed just the SDK that was buggy, not Android. it's ok.